.gitignore

```diff
@@ -34,17 +34,11 @@ docker-compose.override.yml
 ### jupyter
 .ipynb_checkpoints
 
-## File explorers
-**/.DS_Store # OSX Finder
-
```

Comment:
The trailing comment renders this line useless anyways; the `.gitignore` format doesn't support trailing comments, which rendered this line useless. Another line later in this file correctly ignore these files.